addons/calico-vxlan/calico-vxlan.yaml

```diff
@@ -3576,6 +3682,9 @@ spec:
                 configMapKeyRef:
                   name: calico-config
                   key: veth_mtu
+            # Disable AWS source-destination check on nodes.
+            # - name: FELIX_AWSSRCDSTCHECK
+            #   value: Disable
```

Comment:
Are there any cases when users should uncomment this? If there are any, should we document those use cases?

Reply:
This ENV comes from the original YAML, this was an official workaround a bug they had, but fixed, but forgor to remove it from YAML and now it's broken if this ENV is present. See more at: https://github.com/projectcalico/calico/issues/4154#issuecomment-870555270